src/NosCore.PacketHandlers/Command/CreateItemPacketHandler.cs

```diff
@@ -155,13 +156,15 @@ public override async Task ExecuteAsync(CreateItemPacket createItemPacket, Clien
                     case EquipmentType.SecondaryWeapon:
                         wearable.SetRarityPoint();
                         break;
+
```

Comment:
Seems like all those thing would be better in the item creation service not sure why I put it here. Can be moved to replace the other thing using polymorphism over switch will be better